src/jdk.compiler/share/classes/com/sun/tools/javac/code/Symbol.java

```diff
@@ -1786,33 +1786,33 @@ public <R, P> R accept(Symbol.Visitor<R, P> v, P p) {
     public static class RecordComponent extends VarSymbol implements RecordComponentElement {
         public MethodSymbol accessor;
         public JCTree.JCMethodDecl accessorMeth;
-        /* the original annotations applied to the record component
-         */
-        private final List<JCAnnotation> originalAnnos;
+
         /* if the user happens to erroneously declare two components with the same name, we need a way to differentiate
          * them, the code will fail anyway but we need to keep the information for better error recovery
          */
         private final int pos;
 
         private final boolean isVarargs;
 
+        public JCVariableDecl ast;
```

Comment:
I wouldn't let this field, public we would like to at least control that nobody can accidentally overwrite this field, so my proposal is to make it private and let clients read it